tensordict/tensordict.py

```diff
@@ -308,6 +309,11 @@ def wrapper(*args, **kwargs):
 class TensorDictBase(MutableMapping):
     """TensorDictBase is an abstract parent class for TensorDicts, a torch.Tensor data container."""
 
+    LOCK_ERROR = (
+        "Cannot modify locked TensorDict. For in-place modification, consider "
+        "using the `set_()` method and make sure the key is present."
+    )
```

Comment:
This is really neat!

Reply:
Note to self: we should do that for all error msg